apps/wallet/src/ui/app/components/accounts/AccountMultiSelect.tsx

```diff
@@ -34,3 +37,41 @@ export function AccountMultiSelect({
 		</ToggleGroup.Root>
 	);
 }
+
+export function AccountMultiSelectWithControls({
+	selectedAccountIDs: selectedAccountsFromProps,
+	accounts,
+	onChange: onChangeFromProps,
+}: AccountMultiSelectProps) {
+	const [selectedAccounts, setSelectedAccounts] = useState(selectedAccountsFromProps);
+	const onChange = (value: string[]) => {
```

Comment:
Why do we need this? Why not use the `selectedAccountIDs` and rely on the parent updating it? I think we should use account ids in the wallet at least whenever possible

Reply:
The designs call for the addition of a "Select / Deselect All button" which is why we have this additional layer of state

<img width="423" alt="image" src="https://github.com/MystenLabs/sui/assets/122397493/a1954cf3-4ca9-4a92-87d5-6870e71a9315">

Reply:
So we need the extra state `selectedAccounts` so show the select/deselect button? We can do the same with only `selectedAccountIDs`, right?

Reply:
will revisit this in a future PR